Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -45,18 +43,19 @@ public class PlayerActivate : MonoBehaviour
         float clickDelay = 0;
         float clickDelayStartTime = 0;
 
-        public float RayDistance = 0;           // Distance of ray check, tune this to your scale and preference
-        public float ActivateDistance = 2.3f;   // Distance within which something must be for player to activate it. Tune as needed.
+        public float RayDistance = 75f;         // Approximates classic at full view distance (classic's default setting).
+                                                // This is needed for using "Info" mode and clicking on buildings, which can
+                                                // be done in classic as far as the view distance.
 
-        // Maximum distance from which different object types can be activated, in classic distance units
-        public float DefaultActivationDistance = 128;
-        public float DoorActivationDistance = 128;
-        public float TreasureActivationDistance = 128;
-        public float PickpocketDistance = 128;
-        public float CorpseActivationDistance = 150;
-        //public float TouchSpellActivationDistance = 160;
-        public float StaticNPCActivationDistance = 256;
-        public float MobileNPCActivationDistance = 256;
+        // Maximum distance from which different object types can be activated, converted from classic units (divided by 40)
```

Comment:
I wonder if we should use value * MeshReader.GlobalScale here? It's unlikely, but there's always a chance someone might change the global scale later. It would also lead code readers back to the GlobalScale const value in MeshReader with full comments on why we divide by 40.

Reply:
Updated to use ```value * MeshReader.GlobalScale```.

I also set the values as consts, no longer public variables editable from the editor.

The way the public variables editable from the editor were before (in master branch):
RayDistance: Unity distance value. Overridden in the file so had no effect.
Activation distance values: Classic distance value (then multiplied by MeshReader.GlobalScale)

My original idea with the cleanup here (in addition to no longer overriding RayDistance so it would actually have an effect) was to unify these all as unity distances, so the expected value entered from the Unity editor would be used.

To show that MeshReader.GlobalScale is being used, we could unify the values all as classic ones, with anyone changing them having the responsibility to understand they are not the usual Unity distance values, but classic distance values. Seems undesirable.

So, considering the above, and that I figure probably nobody is modifying these properties from the editor anyway, I decided to just make them private variables. Let me know if the current solution is OK with you.